scvi/train/_metrics.py

```diff
@@ -0,0 +1,53 @@
+import torch
+from torchmetrics import Metric
+
+
+class ElboMetric(Metric):
+    def __init__(self, n_obs_total: int, dist_sync_on_step: bool = False):
```

Comment:
typing should be Optional[int] if you are accepting None

Reply:
my suggestion earlier is to add another kwarg here like `mode:str` or `train=True`.